### PR TITLE
Phone number detection

### PR DESCRIPTION
**Pull request by babenek**
## Description

Please include a summary of the change and which is fixed.

- Add international phone format number as PII

## How has this been tested?

Please describe the tests that you ran to verify your changes.

- [x] UnitTest
- [x] TODO: rollback custom benchmark after https://github.com/Samsung/CredData/pull/44
- [x] Benchmark

### CODE REVIEW

File: credsweeper/rules/config.yaml [hunk 1]
```diff
@@ -1,3 +1,15 @@
+- name: Phone
+  severity: info
+  type: pattern
+  values:
+    - (^|[^0-9A-Za-z])(?P<value>\+[1-9][0-9]{6,14})([^=0-9A-Za-z]|$)
```
Comment (csh519) [inline credsweeper/rules/config.yaml:5]: Could you explain it more about `{6,14}`?

As I see the page which you linked(https://en.wikipedia.org/wiki/List_of_mobile_telephone_prefixes_by_country).
The minimum length of `Size of NN` was 4 and maximum was 13.

Comment (babenek) [reply credsweeper/rules/config.yaml:5]: NN- means after country prefix. e.g. 

Algeria  +213 NN= 4 -> full phone +2130000 (7 digits)
first digit in the regex is not 0 [1-9] then we found 6 digits.
max number just decide it should be 15 (maximum in https://gist.githubusercontent.com/ally-commits/9073ff23fc7f96fab1290fdec22775bc/raw/0232c21262e651cd0837105c26e2ad0546b06459/Countries%2520Phone%2520Number%2520Length)

Comment (csh519) [reply credsweeper/rules/config.yaml:5]: Thank you for explanation. I understood well.

File: .github/workflows/benchmark.yml [hunk 2]
```diff
@@ -62,7 +62,8 @@ jobs:
     - name: Checkout CredData
       uses: actions/checkout@v3
       with:
-        repository: Samsung/CredData
+        repository: babenek/CredData
+        ref: phone
```
Comment (babenek) [inline .github/workflows/benchmark.yml:66]:
```suggestion
        repository: Samsung/CredData
```

**Review by Yullia — APPROVED** [review]
Approved

**Review by csh519 — APPROVED** [review]
Thank you for adding new rule `Phone`.

LGTM :+1:

**Pull request merged**